src/crate/theme/rtd/crate/static/css/custom.css

```diff
@@ -1840,3 +1834,9 @@ p.caption:hover > a.headerlink,
 div.code-block-caption:hover > a.headerlink {
   visibility: visible;
 }
+
+/* remove margin from paragraphs in ul & ol li's */
+main ul li p,
+main ol li p {
+  margin: 0;
+}
```

Comment:
Hi. Thanks a stack for those improvements. In my other patch https://github.com/crate/cratedb-guide/pull/53, I was running [such inline CSS directives](https://github.com/crate/cratedb-guide/blob/8dfdafa59c8e89b4ef39ab3bf2ea28de040666dc/docs/feature/cluster/index.md?plain=1#L15-L22), apparently also adjust some undesired rendering behavior.

```
<style>
.field-list dd {
  margin-bottom: 1em !important;
}
.field-list p {
  margin-bottom: 0.5em;
}
</style>
```

Do you think they are related to your improvements, made obsolete by them, and could potentially be removed from the inline spots of that patch, when using the newest and shiniest of our modernizations?

Reply:
@amotl not as of now, since we specifically target `ul` and `ol`'s, not `dl`'s, so none of those would be targeted by the other css.